STATIC_CODE_CHECKS.rst

```diff
@@ -465,18 +465,18 @@ Run the ``ruff`` check for the ``tests/core.py`` file with verbose output:
 
      breeze static-checks --type ruff --file tests/core.py --verbose
 
-Run the ``ruff for the ``tests.core`` package with verbose output:
+Run the ``ruff`` check for the ``tests.core`` package with verbose output:
 
 .. code-block:: bash
 
      breeze static-checks --type ruff --file tests/core/* --verbose
 
-Run the ``black`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
+Run the ``ruff`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
```

Comment:
```suggestion
Run the ``ruff-format`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
```

Ruff [linter](https://docs.astral.sh/ruff/linter/) and [formatter](https://docs.astral.sh/ruff/formatter/) are [different pre-commit hooks](https://github.com/apache/airflow/blob/20d547ecd886087cd89bcdf0015ce71dd0a12cef/.pre-commit-config.yaml#L181-L187). To your earlier point, there is already a `ruff` example above this example so this one should show `ruff-format` since the Ruff formatter is a [drop-in replacement for Black](https://docs.astral.sh/ruff/formatter/#philosophy).